src/entities/Basket/Basket.controller.ts

```diff
@@ -0,0 +1,32 @@
+import StatusCodes from 'http-status-codes';
+import { Request, Response } from 'express';
+import validateBasketReq from './Basket.validation';
+import Basket from './Basket.schema'
+
+const { BAD_REQUEST, CREATED, OK } = StatusCodes;
+
+export const getBasketById = async (req: Request, res: Response) => {
+    try {
+        const basket = await Basket
+            .findById(req.params.id)
+            .populate('createdByUserId', 'name')
+            .populate('targetUserID', 'name')
+        return res.status(OK).json(basket);
```

Comment:
We will need also the books - so please populate them too :)

Reply:
DONE -> 6f7e3908c7bc082288764d24466abc46ce48508f
I need to change schemat to apply 'populate'